pyiode/iode_database/abstract_database.pyx

```diff
@@ -35,6 +35,8 @@ cdef class _AbstractDatabase:
         pass
 
     # Public methods
+
+    # GDM> I do not think this property should exist/be public (isn't the subclass name enough?)
```

Comment:
This property is indeed obsolete. In my first implementation of the IODE database classes, I use Python class names as CommentsDatabase to not mess up with the C++ class names. But then I learned how to create alias names for the C++ classes and then reuse the same names as the C++ class names. 
To make a long story short, I will remove this property. Thanks for the suggestion.